spec/unit/mutant/integration/rspec_spec.rb

```diff
@@ -115,10 +115,20 @@
   end
 
   before do
-    expect(RSpec::Core::ConfigurationOptions).to receive(:new).with(%w[spec --fail-fast]).and_return(options)
-    expect(RSpec::Core::Runner).to receive(:new).with(options).and_return(runner)
-    expect(RSpec).to receive(:world).with(no_args).and_return(world)
-    allow(Time).to receive(:now).and_return(Time.now)
+    expect(RSpec::Core::ConfigurationOptions).to receive(:new)
+      .with(%w[spec --fail-fast])
+      .and_return(rspec_options)
+
+    expect(RSpec::Core::Runner).to receive(:new)
+      .with(rspec_options)
+      .and_return(rspec_runner)
+
+    expect(RSpec).to receive(:world)
+      .with(no_args)
```

Comment:
No, it's mainly for use in methods that receive optional arguments and you want to specify the behaviour of the method when it receives nothing. I've overused it in the past too, and recently begun only asserting it in cases where a method takes optional args only.